Assets/Scripts/Enviroment/EnvironmentBehaviour.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnvironmentBehaviour : MonoBehaviour
+{
+    [Header("Stats")]
+    public float moveSpeed;
+    public float lifeSpan;
+
+    [Header("Despawn when Touching:")] 
+    public bool tree;
+    public bool bush;
+    public bool rock;
+    public bool squiggle;
+    public bool water;
```

Comment:
Moet dit allemaal public zijn of kan het ook gewoon [Serlizedfields] private zijn?